Runner/suites/Multimedia/Audio/AudioRecord/run.sh

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+#!/bin/sh
+
+# Import test suite definitions
+. $(pwd)/init_env
+TESTNAME="audio_record_test"
+TESTBINARY="parec"
+
+#Global variable to hold PID which  needs to be killed at the end of the test
+KILLPID=()
+
+# Import test functions
+. $TOOLS/functestlib.sh
+test_path=$(find_test_case_by_name "$TESTNAME")
+echo "--------------------------------------------------------------------------"
+echo "-------------------Starting $TESTNAME Testcase----------------------------"
+
+echo "Checking if dependency binary is available"
+check_dependencies $TESTBINARY
+
+# Prepare environment
+mkdir -p results/audiotestresult
+chmod -R 777 results/audiotestresult
+
+# Start logs
+dmesg -C
+tail -f /var/log/syslog > results/audiotestresult/syslog_log.txt &
+KILLPID[${#KILLPID[@]}+1]="$!"
+
+dmesg -w > results/audiotestresult/dmesg_log.txt &
+KILLPID[${#KILLPID[@]}+1]="$!"
+
+#Start the Playback
+parec --rate=48000 --format=s16le --channels=1 --file-format=wav /tmp/rec1.wav -d regular0 &
+PID = $!
+KILLPID[${#KILLPID[@]}+1]="$!"
+sleep 10
+
+#Check whether playback started
+if [-z "$PID"]; then
+  echo "Fail to start the test binary $TESTBINARY"
+  exit 1
+else
+  echo "Test Binary $TESTBINARY is running successfully"
+fi
+
+check_audio_pid_alive() {
```

Comment:
this. function is duplicated. Maybe it should go to the library as a general purpose function?